pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/Connection.java

```diff
@@ -46,6 +47,8 @@
   Connection(BrokerSelector brokerSelector, PinotClientTransport transport) {
     _brokerSelector = brokerSelector;
     _transport = transport;
+    // TODO get broker list from brokerSelector
```

Comment:
Can we add a getBrokerList() method to the `BrokerSelector` interface? By doing so, we can get the broker list out of it.

Reply:
That was my plan to take closer look on this in next PR.
Do you think it's better to add it here or create new PR?